src/pages/organize/[orgId]/people/[personId]/index.tsx

```diff
@@ -54,8 +55,11 @@ const PersonProfilePage: PageWithLayout<PersonProfilePageProps> = ({
         </title>
       </Head>
       <Grid container direction="row" spacing={6}>
-        <Grid item md={4}>
-          <PersonDetailsCard {...person} />
+        <Grid item lg={4}>
+          <PersonDetailsCard {...{ person }} />
```

Comment:
This is the same as `<PersonDetailsCard person={person}/>`, right? I think the extra object and destructuring operator makes it more difficult to read.

Same goes for the next card below.